test(map): verify `map.jinja` dump using `_mapdata` state (#203)
saltstack-formulas/template-formula: test/integration/share/README.md

> # InSpec Profile: `share`

Reviewer: This file differs substantially with `test/integration/share/README.md` in other formulas (which are already managed by the `ssf-formula`).  However, the changes here are very useful, so we'd need to update the template to include these.  One formula where this would get a little more confusing is the [libvirt-formula](https://github.com/saltstack-formulas/libvirt-formula/blob/6f1400e0021f16765f774aaecf58eda03fbebb4a/test/integration/share/README.md), where there are multiple libraries.  It would be extremely helpful if we could standardise these so that the template can be updated accordingly.

This shows the implementation of the `share` InSpec [profile](https://github.com/inspec/inspec/blob/master/docs/profiles.md).

> It's goal is to share the libraries between all profiles.

Reviewer: ```suggestion
Its goal is to share the libraries between all profiles.
```

## Libraries

### `system`

> The `system` library provides easy access to system dependents informations:

Reviewer: ```suggestion
The `system` library provides easy access to system dependent information:
```

> - `system.hostname`: return the result of `hostname -s` or `hostnamectl --static` based on the availability of each commans

Reviewer: ```suggestion
- `system.hostname`: return the result of `hostname -s` or `hostnamectl --static` based on the availability of either command
```

> - `system.platform`: take `inspec.platform` and mangle things a bit to be useful

Reviewer: ```suggestion
- `system.platform`: based on `inspec.platform`, modify to values that are more consistent from a SaltStack perspective
```

>   - `system.platform[:family]` provides a family name for Arch

Reviewer: ```suggestion
  - `system.platform[:family]` provide a family name for Arch
```

  - `system.platform[:name]` modify `amazon` to `amazonlinux`

>   - `system.platform[:release]` tweak for Arch and Amazon Linux:

Reviewer: ```suggestion
  - `system.platform[:release]` tweak Arch and Amazon Linux:
```

>     - `Arch` is always `base-later`

Reviewer: ```suggestion
    - `Arch` is always `base-latest`
```

>     - `Amazon Linux` release `2018` became `1`

Reviewer: ```suggestion
    - `Amazon Linux` release `2018` is resolved as `1`
```

>   - `system.platform[:finger]` is just the concatenation of the name and the first release number (except for Ubuntu which gives `20.04` for example)

Reviewer: ```suggestion
  - `system.platform[:finger]` is just the concatenation of the name and the first release number (except for Ubuntu which gives `ubuntu-20` for example)
```

Reviewer: Actually, this isn't only `ubuntu-20` is it?  It will resolve as `ubuntu-20.04`, right?  If that's the case, then perhaps this rewording is better:

```suggestion
  - `system.platform[:finger]` is the concatenation of the name and the major release number (except for Ubuntu, which gives `ubuntu-20.04` for example)
```
